src/jtag/drivers/ftdi.c

```diff
@@ -908,133 +920,137 @@ static void oscan1_set_tck_tms_tdi(struct signal *tck, char tckvalue, struct sig
 	ftdi_set_signal(tck, tckvalue);
 }
 
-static void oscan1_reset_online_activate(void)
+static void cjtag_reset_online_activate(void)
 {
-	/* After TAP reset, the OSCAN1-to-JTAG adapter is in offline and
-	non-activated state.  Escape sequences are needed to bring
-	the TAP online and activated into OSCAN1 mode. */
+	/* After TAP reset, the cJTAG-to-JTAG adapter is in offline and
+	non-activated state. Escape sequences are needed to bring the
+	TAP online and activated into the desidered working mode. */
 
 	struct signal *tck = find_signal_by_name("TCK");
 	struct signal *tdi = find_signal_by_name("TDI");
 	struct signal *tms = find_signal_by_name("TMS");
 	struct signal *tdo = find_signal_by_name("TDO");
 	uint16_t tdovalue;
 
-	static const struct {
-	  int8_t tck;
-	  int8_t tms;
-	  int8_t tdi;
+	static struct {
+		int8_t tck;
+		int8_t tms;
+		int8_t tdi;
 	} sequence[] = {
-	  /* TCK=0, TMS=1, TDI=0 (drive TMSC to 0 baseline) */
-	  {'0', '1', '0'},
-
-	  /* Drive cJTAG escape sequence for TAP reset - 8 TMSC edges */
-	  /* TCK=1, TMS=1, TDI=0 (rising edge of TCK with TMSC still 0) */
```

Comment:
This is all just whitespace changes, right?

Reply:
yes